New example and README.md update

alexmaragko opened the pull request:
This PR includes the following changes/additions:

- New example file is added (example2.py).
- README.md is updated with installation instructions when cloning the repository.
- requirements.txt is updated to include numpy.

alexmaragko committed:
New example file added. README.md update.

alexmaragko committed:
remove matplotlib import in example2.py

alexmaragko committed:
change TypeError to XMLSyntaxError in test_parsers

PAHdb commented:
Can you update the example presented in the README.md-file to be that you provide in example2.py?

alexmaragko committed:
update example in README.md

alexmaragko committed:
examples and transitions updates

alexmaragko commented:
- Swapped names of example files.
- Changed np.exp(val) - 1 to np.expm1(val). The values in the tests agree to more than 20 decimals. 
- Changed the assertion in the tests to np.testing.assert_almost_equal.
- The overflow encounter RuntimeWarning is still present even when changing to np.expm1.
